app/views/hub/forced_password_resets/edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="slab">
+  <div class="grid">
+    <div class="grid__item width-one-half shift-one-fourth">
```

Comment:
The login page has very little left whitespace compared to this page, it might be nice if the two matched. Possibly by getting rid of the grid (could try to match devise/sessions/new)